### PR TITLE
add `backpack:publish-assets` command that publishes CSS and JS assets

### PR DESCRIPTION
**Pull request by tabacitu**
### BEFORE - What was wrong? What was happening before this PR?

In order to publish the CSS and JS assets you had to google how to do it, in order to copy-paste this:

```bash
php artisan vendor:publish --provider="Backpack\CRUD\BackpackServiceProvider" --tag=”public” --force
```

### AFTER - What is happening after this PR?

You can do the above using:

```bash
php artisan backpack:update
```

### How did you achieve that, in technical terms?

Quickly put together a new command that does it.

### Is it a breaking change?

No. But the command name is pretty general.

### How can we test the before & after?

Do `php artisan backpack:update` and it should do the same thing as the other command (publish CRUD assets).

### CODE REVIEW

**Review by pxpm — CHANGES_REQUESTED** [review]
@tabacitu totally agree with the purpose, but 100% against the name. 

`php artisan backpack:assets-publish`
`php artisan backpack:publish-assets`
`php artisan backpack:public-assets`

Any of those are better of `Update` I guess.

**Comment by tabacitu** [comment]
Haha fair enough @pxpm . That's the reason I didn't hit `merge`, I wasn't quite comfortable with `php artisan backapck:update` either.

Ok let's go with `php artisan backpack:publish-assets` then. Will do!

**Comment by tabacitu** [comment]
I renamed it to `php artisan backpack:publish-assets`. Could you please double-check / polish / merge, Pedro?

File: src/app/Console/Commands/PublishAssets.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     */
+    private function runConsoleCommand($command)
+    {
+        $process = new Process($command, null, null, null, 60, null);
+        $process->run(function ($type, $buffer) {
+            if (Process::ERR === $type) {
+                $this->line($buffer);
+            } else {
+                $this->line($buffer);
+            }
```
Comment (pxpm) [inline src/app/Console/Commands/PublishAssets.php:49]:
```suggestion
            $this->line($buffer);     
```

**Pull request merged**